tests/decode-unknown-1/test.yaml

```diff
@@ -1,5 +1,6 @@
 requires:
     min-version: 7
+    lt-version: 8
```

Comment:
Why do we need this ? (deserves a comment maybe)

Reply:
Will add a readme file for this test explaining things.

Reply:
Unresolving until a README is added then